include/wx/msw/private/webrequest_winhttp.h

```diff
@@ -140,6 +143,12 @@ class wxWebRequestWinHTTP : public wxWebRequestImpl
     // the same flag for them as for the server credentials here.
     bool m_tryProxyCredentials = false;
 
+    // Store timeouts, in order to set these before sending request.
+    bool m_isTimeoutsSet = false;
```

Comment:
I'd call this `m_setTimeoutsCalled` which would be more grammatical ("is" and "timeouts" can't be used together). Or maybe I'd just use some special value (`0` if it doesn't have any special meaning for WinHTTP or `-1` maybe) for the timeouts by default and remove it completely.